src/com/xilinx/rapidwright/interchange/PhysNetlistReader.java

```diff
@@ -359,7 +359,19 @@ private static void readRouteBranch(RouteBranch.Reader branchReader, Net net, De
                 Tile tile = device.getTile(strings.get(pReader.getTile()));
                 String wire0 = strings.get(pReader.getWire0());
                 String wire1 = strings.get(pReader.getWire1());
-                PIP pip = new PIP(tile, wire0, wire1);
+                if(tile == null) {

```

Comment:
@litghost  - Yes, this set of APIs will get around that issue:
```
        Integer wire0Idx = tile.getWireIndex(wire0);
        Integer wire1Idx = tile.getWireIndex(wire1);
        PIP pip = tile.getPIP(wire0Idx, wire1Idx);
```

Reply:
See also: http://www.rapidwright.io/javadoc/com/xilinx/rapidwright/device/Tile.html#getPIP-int-int-

Reply:
Right, fixed.